app/windows/Welcome/renderer.jsx

```diff
@@ -27,12 +27,16 @@ class WelcomeWindow extends React.Component {
     window.close()
   }
 
+  handleSubscribe = () => {
```

Comment:
as this is part of the `WelcomePage` logic we should probably move it there.
I think you can have a simple function if you don't want to rewrite it as a class